wildfly-standalone-centos/README.md

```diff
@@ -1,12 +1,12 @@
-# VM-Redhat - WildFly 16.0.0.Final standalone mode
+# Java on Azure | Red Hat WildFly 18.0.1.Final on CentOS 8 (standalone mode)
```

Comment:
Changed the Title to **Red Hat WildFly 18.0.1.Final on CentOS 8 (standalone mode)** because the Azure Quickstart Template has a word Limit of 60 for the Title